lib/signatures.ts

```diff
@@ -3,35 +3,41 @@ import { SignatureHashAlgorithm, SignatureResult } from './types';
 import { getDubHashSignature } from './dubhash';
 import { getUhashSignature } from './uhash';
 import pMap = require('p-map');
+import { join } from 'path';
 
 const { readFile } = promises;
 
 export async function getSignaturesByAlgorithm(
+  baseDir: string,
```

Comment:
I think we can avoid the usage of baseDir here and keep everything as it was.
then have a clean up on filePaths once fileSignatures are generated